roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/install-virtctl.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Create ~/bin
+  ansible.builtin.file:
+    path: ~/bin
+    state: directory
+    mode: "0755"
+
+- name: Get and extract virtctl
+#  become: yes
+  ansible.builtin.unarchive:
+    validate_certs: false
```

Comment:
Possible security issue?

Reply:
@0xFelix are you referring to the "open" file permissions?